template.js

```diff
@@ -46,7 +46,7 @@ if (data.type === 'page_view') {
 }
 
 const user_data = eventData.user_data || {};
-const user_address = user_data.address || {};
+const user_address = user_data.address[0] || user_data.address || {};
```

Comment:
This will cause error "TypeError: undefined is not an object (evaluating 'user_data.address[0]')"
You need to rewrite this to check if user_data.address is array and then only check for first element